cadquery/cq_directive.py

```diff
@@ -278,7 +269,7 @@ class cq_directive_vtk(Directive):
 
     has_content = True
     required_arguments = 0
-    optional_arguments = 2
+    optional_arguments = 0
```

Comment:
Could you explain (I'm a bit rusty on this code)?

Reply:
The directive `cadquery` takes no arguments so it is used as follows:

```rst
.. cadquery::

    import cadquery as cq
```

In the event that a directive accepts arguments it looks something like:

```rst
.. my-directive:: arg1 arg2

    content goes here.
```

Reference:

https://docutils.sourceforge.io/docs/howto/rst-directives.html

> - `required_arguments`: The number of required arguments (default: 0).
> - `optional_arguments`: The number of optional arguments (default: 0).

Reply:
Thanks, clear